cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -663,6 +664,9 @@ func (command *Evaluate) evaluateDocker(ctx *evaluate.Context) (err error) {
 
 // evaluateKubernetes executes the evaluation for each model inside a kubernetes run container.
 func (command *Evaluate) evaluateKubernetes(ctx *evaluate.Context) (err error) {
+	// Define a regex to replace all non alphanumeric characters and "-".
+	kubeNameRegex := regexp.MustCompile(`[^a-zA-Z0-9 -]+`)
```

Comment:
I think we don't want the whitespace in the regex.

```sh
[^a-zA-Z0-9 -]+
[^a-zA-Z0-9-]+
```

Reply:
good catch